### PR TITLE
Implementing Official CVE Feed alpha

### PR DESCRIPTION
**Pull request by PushkarJ**
- Supporting blog to the announcement of the CVE feed
- Covers implementation details, design considerations
- Links to original blog

Preview: https://deploy-preview-330--kubernetes-contributor.netlify.app/blog/2022/09/12/k8s-cve-feed-alpha/
/sig security docs
/area security

related to https://github.com/kubernetes/website/pull/35608 (Need both to be merged together) and https://github.com/kubernetes/enhancements/issues/3203
/hold

### CODE REVIEW

File: content/en/blog/2022/2022-09-12-Implementing-the-Auto-Refreshing-Official-CVE-Feed/index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+Adding the `official-cve-feed` label is restricted to limited number of trusted
+community members. Who has access to add this label is defined
+in [this configuration file](https://github.com/kubernetes/test-infra/blob/master/config/prow/plugins.yaml#L149-L159)
+. Any updates to this configuration file require the changes to go through the
+existing [code review and approval process](/docs/guide/pull-requests/)
+
+#### Prow Configuration
+
+Prow job is defined in
+a `kubernetes/infra` [configuration file](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml#L94-L115)
```
Comment (jberkus) [inline content/en/blog/2022/2022-09-12-Implementing-the-Auto-Refreshing-Official-CVE-Feed/index.md:80]: Er, this is the implementation blog post.  Did you paste the wrong text?

**Comment by jberkus** [comment]
Pushkar,

I've created this hackmd file because I wanted to rewrite parts of your blog post for clearer language, and that's easier than giving you a bunch of individual edits:  https://hackmd.io/-5mXZZQWQyaTbU45O6AHhQ

**Comment by jberkus** [comment]
Done editing the Hackmd file.  Please review it; I think it improves some of the text around describing your implementation.  Particularly, check how I converted passive to active voice in a lot of sentences.

**Review by sftim — COMMENTED** [review]
Thanks. I think we could merge this as-is.

Some feedback / tweaks, if you want them.

/lgtm

**Comment by sftim** [comment]
/hold

due to https://github.com/kubernetes/contributor-site/issues/281

**Comment by PushkarJ** [comment]
@jberkus can you please grant me access to hackmd link you shared?

**Comment by jberkus** [comment]
Pushkar: it should be public. What are you seeing when you try to access it?

**Comment by jberkus** [comment]
Aha, never mind, found the problem.

Use the view link if you don't have a hackmd account: https://hackmd.io/-5mXZZQWQyaTbU45O6AHhQ?view

**Comment by PushkarJ** [comment]
Thanks @jberkus and @sftim for the wonderful input via hackmd. I have updated the PR with the suggested changes!

**Comment by imnmo** [comment]
> /hold
> 
> due to #281

@sftim could you please check this PR: https://github.com/kubernetes/contributor-site/pull/331 fixing the issue for future blog posts with **PublishDate** variable

**Comment by sftim** [comment]
Given that this will merge into _main_ which already has the fix from #331 

/hold cancel

:crossed_fingers:

**Pull request merged**